apps/server/src/modules/authentication/strategy/x-api-key.strategy.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { UnauthorizedException } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { Configuration } from '@hpi-schul-cloud/commons/lib';
+import { ConfigService } from '@nestjs/config';
+import { createMock } from '@golevelup/ts-jest';
+import { XApiKeyStrategy } from './x-api-key.strategy';
+import { IXApiKeyConfig } from '../config/x-api-key.config';
+
+describe('XApiKeyStrategy', () => {
+	let module: TestingModule;
+	let strategy: XApiKeyStrategy;
+	let configService: ConfigService<IXApiKeyConfig, true>;
+	Configuration.set('ADMIN_API__ALLOWED_API_KEYS', '1ab2c3d4e5f61ab2c3d4e5f6');
```

Comment:
Please don't set this var "globally" in the `Configuration` module, do it only for this testing suite.

Reply:
This can be deleted I think forgot to delete that